src/main/java/org/gridsuite/modification/server/entities/equipment/modification/LoadModificationEntity.java

```diff
@@ -34,19 +34,19 @@ public class LoadModificationEntity extends InjectionModificationEntity {
     @Enumerated(EnumType.STRING)
     private OperationType loadTypeOp;
 
-    @Column(name = "activePowerValue")
-    private Double activePowerValue;
+    @Column(name = "p0_value")
```

Comment:
this  @Column(name = "loadTypeValue") will generate a column in db named load_type_value which is not the case for p0_value because we have the "0" as a separator